src/PockyBot.NET/Services/Triggers/Ping.cs

```diff
@@ -0,0 +1,27 @@
+using GlobalX.ChatBots.Core.Messages;
+using Microsoft.Extensions.Options;
+using PockyBot.NET.Configuration;
+using PockyBot.NET.Constants;
+using PockyBot.NET.Persistence.Repositories;
+
+namespace PockyBot.NET.Services.Triggers
+{
+    internal class Ping : Trigger
+    {
+        public Ping(IOptions<PockyBotSettings> settings, IPockyUserRepository pockyUserRepository) : base(settings, pockyUserRepository) { }
+
+        protected override string Command => Commands.Ping;
+
+        protected override bool DirectMessageAllowed => true;
+
+        protected override bool CanHaveArgs => false;
+
+        protected override bool PermissionsRequired => false;
+
+        public override Message Respond(Message message) {
+            return new Message {
+                Text = "pong. I'm alive!"
```

Comment:
shouldn't this have a version number in the response

Reply:
Which version are we talking? The library version? A version provided to the bot via environment variable? Perhaps we can add that later? I'm trying to keep this PR stripped down so it's easier to review

Reply:
I would have it pull an environment variable that it sets, but that can be overwritten by the project that implements the library? but yeah, maybe add in a later PR